bin/bench-tx/src/execution_benchmarks/mod.rs

```diff
@@ -0,0 +1,20 @@
+use core::fmt;
+
+pub mod utils;
+
+/// Indicates the type of the transaction execution benchmark
+pub enum ExecutionBenchmark {
+    ConsumeSingleP2ID,
+    ConsumeMultipleP2ID,
+    CreateSingleP2ID,
+}
+
+impl fmt::Display for ExecutionBenchmark {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            ExecutionBenchmark::ConsumeSingleP2ID => write!(f, "consume single P2ID note"),
+            ExecutionBenchmark::ConsumeMultipleP2ID => write!(f, "consume multiple P2ID notes"),
```

Comment:
```suggestion
            ExecutionBenchmark::ConsumeTwoP2ID => write!(f, "consume two P2ID notes"),
```

Nit: I think I would specify the number in the name; makes it easier to understand what the benchmark is doing without checking the exact setup.